Posts/2021/03/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows.md

```diff
@@ -0,0 +1,117 @@
+---
+post_title: Lightning Fast and Easy Provisioning of Git with SSH Key Authentication on Windows
+username: DarwinJS@sanoys.com
+Catagories: PowerShell
+tags: Git, DevOps
+Summary: Getting your team setup with Git over SSH as quickly as possible!
+featured_image: ./media/DarwinJS-Contribution-LightningFastandEasyProvisioningofGitwithSSHKeyAuthenticationonWindows/windows-git-ssh.png
+CanonicalURL: https://missionimpossiblecode.io/post/lightning-fast-and-easy-provisioning-of-git-with-ssh-key-authentication-on-windows/
+---
+
+
+Maybe you have a team of Windows developers that are onboarding for your new Git server installation or maybe you've decided to drop http password authentication to your existing Git server (due to it's many problems).  Your next steps may well be into a rough and rocky rabbit hole when you were holding out hope for simplicity (you know the kind you've fallen into before if you've been in tech for more than about 45 minutes).
+
+The guides on the internet for getting Windows setup for SSH authentication for Git are unnecessarily complex.
+
+My inner tool smith really loathes when the very first steps into something new are fraught with rocky rabbit holes - so I took on the challenge of creating an easier way. 
+
+The resultant tool is a 20 line PowerShell script that deploys Git, configures SSH and leaves the public key on your clipboard so you can paste it into GitLab or any other Git collaborative webserver. There is also an optional connectivity test.
+
+<!-- more -->
+
```

Comment:
We don't support the summary views in our feed. So this is not used.

Reply:
Removed in another commit!